### ISSUE TITLE
Save states between flights

### ISSUE THREAD

**Issue by gilbertohasnofb** [HHS81/c182#109]
One of my favourite features of the c172p is that when you activate the save state toggle in the aircraft menu, every toggle and switch and knob and lever remain in the same position next time the sim is loaded. It brings a great deal of realism to the sim, and I would say we should consider doing the same for ours.

**Comment by HHS81** [comment]
yep, for sure. The same about fuel state, battery state etc.....

**Comment by hbeni** [comment]
d7ba4fc adds many options to the save-state of the aircraft; i think i got the most items already.
What is not working right now is the state of the avionics busses and battery switches  - are they overridden in some nasal script?

**Comment by HHS81** [comment]
They are set in `c182s-electrical.nas`

**Comment by hbeni** [comment]
Should be fixed in PR #167 (https://github.com/HHS81/c182s/pull/167#issuecomment-331739868) already :)

**Comment by hbeni** [comment]
Open from the pull request is the ground equipment/aircraft securing

**Comment by hbeni** [comment]
In my experiments, aircraft securing now also works. What still does not work is ground equipment like the ladder etc.
